dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node
```

Comment:
Usually called `Dockerfile` instead of `dockerfile`.
Also I would pick a version of Node here (probably 16) `node:16-alpine` and Alpine for smaller image sizes.